press/press/doctype/server/server.py

```diff
@@ -1026,6 +1026,26 @@ def archive(self):
 			frappe.throw(
 				_("Cannot archive server with benches. Please drop them from their respective dashboards.")
 			)
+
+		if frappe.db.exists(
+			"Press Job",
+			{
+				"job_type": "Archive Server",
+				"server": self.name,
+				"server_type": self.doctype,
+				"status": "Success",
+				"creation": (">", frappe.utils.add_to_date(None, days=-1)),
+			},
+		):
+			frappe.throw(_("Archive Server job already completed successfully recently."))
```

Comment:
Maybe just show a message instead of throwing error and return.

Doesn't need filter on `creation`. As we don't reuse the record for re-creating servers.